spi/identity-hub-spi/build.gradle.kts

```diff
@@ -8,4 +9,13 @@ val jacksonVersion: String by project
 dependencies {
     api("org.jetbrains:annotations:${jetBrainsAnnotationsVersion}")
     implementation("com.fasterxml.jackson.core:jackson-databind:$jacksonVersion")
+}
+
+publishing {
```

Comment:
not needed at this point but should plan to include `signing` plugin for publishing to Maven Central.

Also will require tasks for including javadoc, source jars  and signing, can also be done at root level